### DISCUSSION TITLE
Hide a stashed window... When the cursor leaves the window area [Immediately/0.1-1s]

### DISCUSSION THREAD

**Discussion by kerryj89** [rxhanson/RectanglePro-Community#567] [Ideas]
Hello!

Can you consider adding a timer dropdown for "Hide a stashed window... When the cursor leaves the window area"?  I think it might help for accidental overshoot when trying to click on a tab.

Thanks!

**Comment by rxhanson** [comment]
Even though this one came first, closing it as a duplicate of #716. Apologies that it took so long to address!
